content/sd/0003-tempo-e-sincronizacao.md

```diff
@@ -218,16 +253,47 @@ $$
 Cada processo ajusta o seu relógio, de acordo com a diferença recebida.
 
 O algoritmo apresentado foi simplificado, na versão real é tido em conta o
-tempo de transmissão, da forma como foi feito no Algoritmo de Cristian.
+tempo de transmissão, da forma como foi feito no Algoritmo de Cristian. É ainda definido um valor máximo para o RTT (dependendo da exatidão desejada), permitindo ao coordenador eliminar ocasionais leituras que ultrapassem esse valor.
```

Comment:
Nit: try to keep lines wrapped to 80-100 characters max.
Ideally, start every sentence in a different line.